1295. Find Numbers with Even Number of Digits

```diff
@@ -0,0 +1,11 @@
+class Solution(object):
+    def findNumbers(self, nums):
+        """
+        :type nums: List[int]
+        :rtype: int
+        """
+        count = 0 
+        for i in nums:
```

Comment:
nitpick: I'd rather name this variable `num` instead of` i` here. i is traditionally an integer index, in this case ur doing a for each loop.

Reply:
makes sense :)